README.md

```diff
@@ -100,7 +100,13 @@ To run the project, execute the following command:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
+> Mention all of the collaborators of this project..
```

Comment:
- [ ] Kindly, include a "Tests" section on your readme and specific instructions about how to run them properly.  Since there are only integration tests, the command that worked for me was `rake rswag`. 👍